client/src/main/java/tds/student/Student.java

```diff
@@ -0,0 +1,67 @@
+package tds.student;
+
+/**
+ * Represents a student taking an exam
+ */
+public class Student {
+    private long id;
+    private String studentId;
+    private String stateCode;
+    private String clientName;
+
+    /**
+     * @return numeric id for the student
+     */
+    public long getId() {
+        return id;
+    }
+
+    /**
+     * @param id numeric id for the student
+     */
+    public void setId(long id) {
+        this.id = id;
+    }
+
+    /**
+     * @return the external SSID in the ART system
+     */
+    public String getStudentId() {
```

Comment:
something to be careful of here.  a student has an SSID and an External SSID (sometimes called Alternate SSID) and in our deployment we have it setup to use the External SSID as the identifier when the student logs in, but it is a configuration option (I am 95% sure of this) and can be set to use the SSID for the login.  I'm not sure how it gets stored in the tables and what you are needing it for, but you might need to have a getExternalStudentId() and a getStudentId()?